docs/organizations/overview.mdx

```diff
@@ -5,26 +5,107 @@ description: Learn about how to create and manage Clerk organizations and their
+
+## Active organization
+
+There are two types of accounts that a user can have in a Clerk application: a personal account and an organization account. A personal account is the default account type, and it is created when a user signs up for your application. An organization account is created when a user creates or joins an organization within your application.
+
+When a user is a member of an organization, they can switch between their personal account and their organization account. The organization account that a user is currently viewing is called the active organization. The active organization determines which organization-specific data the user can access and which roles and permissions they have within the organization.
+
+When a user signs in to your application, they are viewing their personal account and no organization is set as active. Even if they are a member of only one organization, they must explicitly switch to that organization to set it as active.
+
+The easiest way to allow users to switch between their personal account and their organization account is to use Clerk's [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) component. Once a user selects an organization, that organization is set as the active organization.
+
+## Organization management
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations).
+2. In the navigation sidebar, select **Organizations**. Here, you can view and manage all organizations within your application - both those created by you and those create by your users.
+3. Select a specific organization to view its details, members, and settings. Here, you can update the organization's name, slug, and logo. You can also set the organization's [membership limit](#how-many-organizations-can-i-create) and public and private metadata.
+
+### User membership management
+
+Clerk provides a set of prebuilt components to help your users manage their organization memberships:
+
+- [`<CreateOrganization />`](/docs/components/organization/create-organization) - A form for a user to create a new organization.
+- [`<OrganizationProfile />`](/docs/components/organization/organization-profile) - A profile page for the user's currently active organization. Out of the box, this component's **General** tab displays the organization's information and the **Leave organization** option. The **Members** tab shows the organization's members along with their join dates and roles.
+- [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) - A dropdown menu that allows a user to switch between their personal account and their organization account(s).
+- [`<OrganizationList />`](/docs/components/organization/organization-list) - A list of organizations that a user is a member of.
+
+### Admin membership management
```

Comment:
What are these about? Are they Dashboard operations or you mean the available Backend API operations?